src/styles/mainClassDefinitions.less

```diff
@@ -462,7 +462,7 @@
 	text-align: center;
 	margin-bottom: 10px;
 	margin-right: 5px;
-	width: 100%;
+	margin-left: 5px;
```

Comment:
As discussed, we need to consult with design to see if we'd rather have missing horizontal focus borders or misaligned elements. Either way, we need to deal with a tradeoff and we try not to make these decisions independently as they will no longer match redlines.